src/config.ts

```diff
@@ -49,9 +49,15 @@ export interface SwaggerConfig {
 
   /**
    * API host, expressTemplat e.g. localhost:3000 or myapi.com, use null for relative urls
+   * OAPI2 compliant
    */
   host?: string | null;
 
+  /**
+   * API hosts array, e.g. [https://locahost:3000, https://myapi.com]
+   */
+  servers?: OpenAPI3Server[];
```

Comment:
Should this be `hosts`?

Reply:
No, should be servers since OAPI3 calls it servers and it's a difference interface.
Hosts don't exist any more, I don't want to support it either.

Reply:
Ah, ok, was confused by the comment above and the check for `this.config.hosts`.. Maybe I'm not reading this correctly.